lib/Sema/SemaExpr.cpp

```diff
@@ -5692,7 +5692,7 @@ ExprResult Sema::ActOnCallExpr(Scope *Scope, Expr *Fn, SourceLocation LParenLoc,
       }
       // In an unchecked scope, pass void types as the type arguments.
       if (DeclRefExpr *DR = dyn_cast<DeclRefExpr>(Fn)) {
-        SmallVector<DeclRefExpr::GenericInstInfo::TypeArgument, 4>
+        SmallVector<TypeArgument, 4>
```

Comment:
Please move typeArgumentInfos to this line.

Reply:
Done.